Rock/Model/LocationService.Partial.cs

```diff
@@ -42,29 +41,15 @@ public partial class LocationService
         /// <returns>
         /// The first <see cref="Rock.Model.Location" /> where an address match is found, if no match is found a new <see cref="Rock.Model.Location" /> is created and returned.
         /// </returns>
-        public Location Get( string street1, string street2, string city, string state, string postalCode, string country, bool verifyLocation = true )
+        public Location Get( string street1, string street2, string city, string state, string postalCode, string country, bool verifyLocation = true, Group group = null )
         {
```

Comment:
@chrismcgrath Same comment as other pull request. This is a breaking change, but unfortunately with the existing optional `verifyLocation` param, no way to avoid that, but can you convert to use method overrides so that future changes can be done without it being a breaking change.